### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by bhigy**
I improved the instructions in the README and adjusted small details to make things easier for people starting with our code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,41 +14,92 @@ pip install torchvision
...
+You can now train a model using one of the examples provided under
+`platalea/experiments`, e.g.:
 
 ```sh
-python experiments/basic-stack/run.py
+cd platalea/experiments/flickr8k
+mkdir -p runs/test
+cp basic.py runs/test
+cd runs/test
+python basic.py
```
Comment (egpbos) [inline README.md]: ~I think it should be `python basic_default.py`. I'm all in favor of renaming that to `basic.py` btw :)~ I see you renamed it, good!

Apart from this, though, there is a more important comment to make here: while it is still possible to run the scripts from the source directory, when you have installed the source, you don't actually need to know the source directory anymore. You can just run with `python -m platalea.experiments.flickr8k.basic` from any directory you like. That also saves you the trouble of having to make a copy.

File: platalea/experiments/config.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import configargparse
-from pathlib import Path
 import logging
+import os
```
Comment (egpbos) [inline platalea/experiments/config.py]: Why is `os` necessary?

**Review by egpbos — CHANGES_REQUESTED** [review]
In general, it really helps diff-ability of text files (latex as well) if you write one sentence per line. That way, when you change one sentence, it is immediately apparent in the diff, which also makes reviewing changes easier. Using an arbitrary max line length to break on just complicates editing.

Other than that (and the two comments): looks good to me!

File: README.md (hunk 1 above)
Comment (bhigy) [reply README.md]: Updated

File: platalea/experiments/config.py (hunk 2 above)
Comment (bhigy) [reply platalea/experiments/config.py]: Good catch.

**Pull request merged**